workspaces/adoption-insights/plugins/adoption-insights/src/components/AdoptionInsightsPage/AdoptionInsightsPage.tsx

```diff
@@ -37,7 +37,7 @@ export const AdoptionInsightsPage = () => {
   return (
     <DateRangeProvider>
       <Page themeId="home">
-        <InsightsHeader title="Insights" />
+        <InsightsHeader title="Adpotion Insights" />
```

Comment:
Typo
Adoption Insights
```suggestion
        <InsightsHeader title="Adoption Insights" />
```